modules/frontend/metrics_query_range_sharder.go

```diff
@@ -253,18 +254,18 @@ func (s *queryRangeSharder) buildBackendRequests(ctx context.Context, tenantID s
 				Step:      step,
 				QueryMode: searchReq.QueryMode,
 				// New RF1 fields
-				BlockID:          m.BlockID.String(),
-				StartPage:        uint32(startPage),
-				PagesToSearch:    uint32(pages),
-				Version:          m.Version,
-				Encoding:         m.Encoding.String(),
-				Size_:            m.Size,
-				FooterSize:       m.FooterSize,
-				DedicatedColumns: dc,
-				Exemplars:        exemplars,
+				BlockID:       m.BlockID.String(),
+				StartPage:     uint32(startPage),
+				PagesToSearch: uint32(pages),
+				Version:       m.Version,
+				Encoding:      m.Encoding.String(),
+				Size_:         m.Size,
+				FooterSize:    m.FooterSize,
+				// DedicatedColumns: dc, for perf reason we pass dedicated columns json in directly to not have to realloc object -> proto -> json
```

Comment:
what about adding a new property DedicatedColumnsJSON to the QueryRangeRequest and deprecating the existing one?

Reply:
I know it's used everywhere :P

Reply:
i like this change, but the most difficult part would be the rollout. the query frontends would roll first and start sending the new format which the queriers would not know how to interpret.

that doesn't mean we can't do it, but it does mean we should be thoughtful if the cost of manually rolling tempo cells is worth it.

imo we are asking questions about serverless now and how much/what data needs to be passed from frontend -> querier. if we make a big shift that requires a manual rollout we should do it along with deciding what the relationship between querier and frontend is.